### ISSUE TITLE
Add links in the issues to reference docs

### ISSUE THREAD

**Issue by ericwb** [vmware/precaution#65]
**Is your feature request related to a problem? Please describe.**
When running precaution, sometimes the user may not fully understand the issue being flagged.  That's where it would be very useful to have "more info" type links.

**Describe the solution you'd like**
For example:
```
 B413:blacklist

The pyCrypto library and its module DSA are no longer actively maintained and have been deprecated. Consider using pyca/cryptography library.
```
I'd like to see a link on B413:blacklist to the actual Bandit documentation for such error. Like:
https://bandit.readthedocs.io/en/latest/blacklists/index.html

In the output of Bandit, it will generate a `more_info` key that is an HTTP link to the reference documentation.  So we should be able to use that.  

Not sure if Gosec provides the same, but worth looking into.

**Describe alternatives you've considered**
none

**Additional context**
https://github.com/ericwb/bandit/pull/2/checks

**Comment by MVrachev** [comment]
I created a pull request to add "more info" on the issues found by Bandit.

Gosec at least for now doesn't have a good documentation about the issues it finds.

Here is a thread about this topic in which I participated:
https://github.com/securego/gosec/issues/127

**Comment by ericwb** [comment]
What about the `raw_details` field of the annotation object?

https://developer.github.com/v3/checks/runs/#annotations-object

**Comment by MVrachev** [comment]
I tried with raw_details. First with markdown: 

![image](https://user-images.githubusercontent.com/16246778/50207237-90179680-0376-11e9-872d-b80b24db23cd.png)

Then with html: 
![image](https://user-images.githubusercontent.com/16246778/50207289-b50c0980-0376-11e9-891a-4430a8c1c25a.png)

This is how I put the html tag: 
![image](https://user-images.githubusercontent.com/16246778/50207305-bfc69e80-0376-11e9-8159-94d00fbb3e8f.png)

**Comment by MVrachev** [comment]
I tried the same thing with message/title:

Message/title with markdown: 
![image](https://user-images.githubusercontent.com/16246778/50207551-79be0a80-0377-11e9-8a97-4b41a522936b.png)

Message/title HTML: 
![image](https://user-images.githubusercontent.com/16246778/50207718-eb965400-0377-11e9-84e6-eacea3c8eac6.png)

**Comment by evqna** [comment]
@MVrachev The [checks run documentation](https://developer.github.com/v3/checks/runs/#output-object-1) says markdown is supported by the **summary** and **text** fields of the output object, did you try those?

**Comment by MVrachev** [comment]
We can always add web-links in the summary and text fields.
But those are general fields.
We wanted to have a weblink to every specific issue.

**Comment by MVrachev** [comment]
The other think we can do is to scan which Bandit issues we have and give web-links only to them.

**Comment by evqna** [comment]
Yes, maybe list every issue that was found (possibly with a counter for the number of occurrences) and make the issue name link to the documentation.

**Comment by MVrachev** [comment]
At least for Bandit we provided links in the issues to reference docs.
For Gosec there is no good documentation for this. I hope in future there will be.
Should we close this?

**Comment by ericwb** [comment]
We need something similar for gosec, so we should leave this open.
